src/material-experimental/mdc-button/_mdc-button.scss

```diff
@@ -1,13 +1,86 @@
+@import '@material/button/mixins';
 @import '../mdc-helpers/mdc-helpers';
 
+$_mat-button-ripple-opacity: 0.1;
+
 @mixin mat-button-theme-mdc($theme) {
+  $primary: mat-color(map-get($theme, primary));
+  $accent: mat-color(map-get($theme, accent));
+  $warn: mat-color(map-get($theme, warn));
+
   @include mat-using-mdc-theme($theme) {
-    // TODO: MDC theme styles here.
+    @include mdc-button-without-ripple($query: $mat-theme-styles-query);
+  }
+
+  .mat-mdc-button {
+    // The ripple element is created from the RippleRenderer rather than the MDC ripple, so it is necessary to provide
+    // the color and opacity styling manually.
+    .mat-ripple-element {
+      opacity: $_mat-button-ripple-opacity;
+      background-color: currentColor;
+    }
   }
 }
 
 @mixin mat-button-typography-mdc($config) {
-  @include mat-using-mdc-typography($config) {
-    // TODO: MDC typography styles here.
+  @if config {
```

Comment:
I updated the `mat-using-mdc-typography` mixin, now you don't need the if, just put the styles once, e.g.

```scss
@include mat-using-mdc-typography($config) {
  @include mdc-button-without-ripple($query: $mat-typography-styles-query);
}
```